Firestore/Source/Model/FSTDocumentSet.mm

```diff
@@ -30,7 +33,7 @@
  * The type of the index of the documents in an FSTDocumentSet.
  * @see FSTDocumentSet#index
  */
-typedef FSTImmutableSortedDictionary<FSTDocumentKey *, FSTDocument *> IndexType;
+typedef std::map<DocumentKey, FSTDocument *> IndexType;
```

Comment:
Nit: `using` is preferred to `typedef` for new code. OTOH, I guess argument could be made for consistency with existing code.